pages/styles/style-gallery.css

```diff
@@ -0,0 +1,289 @@
+h1 {
+    color: rgb(242, 232, 207);
+    text-align: center;
+}
+
+  nav ul { 
+    margin: 0; 
+    padding: 5px; 
+    list-style-type: none; 
+    text-align: center; 
+    background-color: rgb(56, 102, 65); 
+    } 
+   
+  nav ul li {  
+    display: inline; 
+      font-size: 15px;
+    } 
+   
+  nav ul li a { 
+    text-decoration: none; 
+    padding: .2em 1em; 
+    color: #fff; 
+    background-color: rgb(56, 102, 65); 
+    } 
+   
+  nav ul li a:hover { 
+    color: #000; 
+    background-color: #fff; 
+    } 
+
+nav {
```

Comment:
[nit] Try to get spacing consistent so that the rules line up visually. When you indent, your readers may think the indented line is intended to be nested! (You may encounter SASS in industry, where nesting rules in a CSS-like file is actually allowed.)